readme.md

```diff
@@ -79,6 +79,8 @@ $ moviebox --help
     $ moviebox -m 2874 -r 3 -p --interactive
 ```
 
+To see all movies with corresponding ID's, take a look [at this list](moviebox/dataset/movie-titles.csv).
```

Comment:
Nice idea! I would indeed be very helpful to have a list with all movie titles and their IDs linked to the main `readme` document, though it be would even better if that document was a `markdown` file instead of a `cv` one. Movie titles and IDs could be placed in a table-like format, e.g:

Movie | ID
------- | ------- 
Movie A | 0
Movie B | 1

Here is a quick and simple [guide](https://guides.github.com/features/mastering-markdown) on how to create tables with markdown. Also, the file can be placed directly under the root directory of the repo, e.g:

```
To see all movies with their corresponding IDs, take a look [at this list](movie-titles.md).
```